packages/app-desktop/gui/NoteEditor/utils/useMarkupToHtml.ts

```diff
@@ -58,9 +63,16 @@ export default function useMarkupToHtml(deps: HookDependencies) {
 			codeHighlightCacheKey: 'useMarkupToHtml',
 			settingValue: deps.settingValue,
 			whiteBackgroundNoteRendering,
+			itemIdToUrl: (id: string, urlParameters = '') => {
+				if (!(id in resources) || !resources[id]) {
+					return null;
+				}
+
+				return resourceFullPath(resources[id].item, resourceBaseUrl) + urlParameters;
+			},
```

Comment:
This fixes the PDF viewer issue &mdash; the renderer previously used `ResourceModel.fullPath(...)` for media attachments (ignoring `resourceBaseUrl`). If present, however, the media renderer prefers `itemIdToUrl` over `Resource.fullPath`.